Form/Type/Filter/ChoiceType.php

```diff
@@ -69,31 +69,30 @@ public function getBlockPrefix()
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $choices = array(
-            self::TYPE_CONTAINS => 'label_type_contains',
-            self::TYPE_NOT_CONTAINS => 'label_type_not_contains',
-            self::TYPE_EQUAL => 'label_type_equals',
+            'label_type_contains' => self::TYPE_CONTAINS,
+            'label_type_not_contains' => self::TYPE_NOT_CONTAINS,
+            'label_type_equals' => self::TYPE_EQUAL,
```

Comment:
Yes, this will avoid to refactor this when dropping support for Symfony <= 2.7 ;)